zipline-bytecode/src/test/kotlin/app/cash/zipline/bytecode/ApplySourceMapToBytecodeTest.kt

```diff
@@ -213,11 +212,16 @@ class ApplySourceMapToBytecodeTest {
       |    });
       |}
       |
-""".trimMargin()
+      """.trimMargin()
 
     val bytecode = quickJs.compile(js, "demo.js")
     val bytecodeWithSourceMap = applySourceMapToBytecode(bytecode, emptySourceMap)
     quickJs.execute(bytecodeWithSourceMap)
     assertThat(quickJs.evaluate("doubleToDisplayString('1000.0')")).isEqualTo("1000.0")
   }
+
+  @Suppress("INVISIBLE_MEMBER") // Access :zipline internals.
```

Comment:
Hiding `internal` things might be more trouble than it’s worth

Reply:
(Though perhaps adding a testing facet gets me where I want to be.)

Reply:
Agreed